modules/vcf/templates/annotate.sh

```diff
@@ -224,6 +224,54 @@ vep() {
   ${CMD_VEP} "${args[@]}"
 }
 
+viab(){
```

Comment:
Add a doc note with link to new BCFtools GitHub issue that mentions that fill-tags is not working for multi-allelics so that we know that we can replace it in the future